src/PrettyFormatter.go

```diff
@@ -48,15 +47,24 @@ func (prettyFormatter *PrettyFormatter) addEntry(key string, value string, table
 
 func (prettyFormatter *PrettyFormatter) addKey(key string) {
 	if !prettyFormatter.keys[key] {
-		prettyFormatter.orderedKeys[len(prettyFormatter.orderedKeys)] = key
+		index := len(prettyFormatter.orderedKeys)
+		log.Debugf("adding key : %s at [%d]\n", key, index)
+		prettyFormatter.orderedKeys[index] = key
 		prettyFormatter.keys[key] = true
 	}
 }
 
-func (prettyFormatter *PrettyFormatter) format(planet Planet) {
+func (prettyFormatter *PrettyFormatter) createSetForPlanet(planet Planet) {
 	var completeTable = make(map[string]string)
-	prettyFormatter.addMetadata(completeTable, planet)
+	address := fmt.Sprintf("%s@%s", planet.user, planet.host)
+	number := strconv.Itoa(planet.outputStruct.position)
+	prettyFormatter.addEntry("Nr.", number, completeTable)
+	prettyFormatter.addEntry("Planet-ID", planet.id, completeTable)
+	prettyFormatter.addEntry("Planet-Name", planet.name, completeTable)
+	prettyFormatter.addEntry("Planet-Address", address, completeTable)
+	prettyFormatter.addEntry("Planet-Type", planet.planetType, completeTable)
```

Comment:
Ist der Aufwand zum Fixen der Tests so hoch, dass es in ein extra Ticket ausgelagert werden muss?

In der [bintest/use_cases_test.rb](https://github.com/appPlant/goo/blob/master/bintest/use_cases_test.rb) sehe ich Tests, die von fixen Spaltenbreiten ausgehen.